graphql.services.yml

```diff
@@ -123,3 +123,9 @@ services:
     arguments: ['@config.storage', '@?plugin.manager.language_negotiation_method']
     tags:
       - { name: config.factory.override, priority: -253 }
+
+  # Query map provider using the composite pattern.
+  graphql.query_provider:
```

Comment:
We might want to make query providers plugins that one can attach to a schema/server.

Reply:
Think e.g. about Search API and how it makes indexes extendable with facets, processors/transformers, etc.